.github/workflows/python-package.yml

```diff
@@ -34,6 +34,16 @@ jobs:
     - name: Install deps
       run: |
         python -m pip install pytest coverage coveralls "Django~=${{ matrix.django-version }}.0"
+    - name: Run migration check
```

Comment:
I'm concerned with the fact that this check depends on demo, depending on a certain Django version output, which may change.

Reply:
@idlesign You are right, that this checks depends on the output. I will rewrite it to use `--check` parameter.
But I am not sure how can I not use `demo` project to run `manage.py`.

Reply:
> But I am not sure how can I not use demo project to run manage.py.

`manage.py` is just a file with instructions, which can be used by themselves in tests for example, or it (the file) can be generated, and again, used in tests.

In fact I think it'd be more useful to implement such a check in `pytest-djangoapp` as a `check_migrations` fixture, for example. In that way it could be used virtually by any app using `pytest-djangoapp`.

Reply:
According to my experience with other project, it often happened, that the ``demo`` was not updated to the newest version of Django and stopped working even if all tests were passing.
So I would consider the dependency on the ``demo`` beneficial, because it would partly also test it.
Of course more systematic approach would be to have separate ``demo`` tests, but I don't see any big disadvantage in having tests dependent on ``demo``.

Anyway, I will try to try to do it as you suggested.